src/index.js

```diff
@@ -158,21 +158,9 @@ function jumpToPast (history, index) {
 }
 // /jumpToPast
 
-// wrapState: for backwards compatibility to 0.4
-function wrapState (state) {
-  return {
-    ...state,
-    history: state
-  }
-}
-// /wrapState
-
 // updateState
 function updateState (state, history) {
```

Comment:
Yes, `wrapState` is definitely not needed. I'm talking about `updateState` - is that really needed anymore?